open_academy/controllers/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove this file also remove the controllers folder. When you need this you'll add it.